.github/workflows/stall.yml

```diff
@@ -0,0 +1,23 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
+          stale-issue-message: 'This issue is stale because it has been open 30 days with no activity. Remove stale label or comment or this will be closed in 5 days.'
```

Comment:
```suggestion
          stale-issue-message: 'This issue is stale because it has been open 30 days with no activity. Remove stale label or comment, otherwise, this issue will be closed in 5 days.'
```